### ISSUE TITLE
Crash when accessing temperature settings (config not initialized properly)

### ISSUE THREAD

**Issue by logix2** [Fushko/gammy#22]
Using Gammy 0.9.56 on Ubuntu 19.10 (with Qt 5.12.4), the application crashes when clicking the second '...' button (the one next to Temperature). This is the error I'm getting:

```
$ gammy
2020-02-17 12:39:02.330 INFO  [28091] [getConfigPath@148] Config path: /home/logix/.config/gammy
2020-02-17 12:39:02.330 DEBUG [28091] [read@120] Settings parsed
terminate called after throwing an instance of 'nlohmann::detail::type_error'
  what():  [json.exception.type_error.302] type must be number, but is null
Aborted (core dumped)
```

**Comment by Fushko** [comment]
It's a config error. Simply delete the file in ~/.config/gammy and let it regenerate.

**Comment by Fushko** [comment]
Should be fixed in v0.9.57. Let me know otherwise.

**Comment by logix2** [comment]
That fixed it, thank you!
